site/app/libraries/database/DatabaseQueries.php

```diff
@@ -3846,6 +3846,16 @@ public function getCourseForUserId($user_id, bool $archived = false): array {
         return $return;
     }
 
+    /**
+     * Get all courses where the user with the specified user_id is assigned as an instructor
+     * @param string $user_id
+     * @return array
+     */
+    public function getInstructorLevelAccessCourse(string $user_id): array {
+        $this->submitty_db->query("SELECT DISTINCT semester, course FROM courses_users WHERE user_id=? AND user_group=1", [$user_id]);
```

Comment:
Why select distinct? This table has a defined primary key on semester, course, and user_id so there is no way for there to be multiple identical entries per user.